### DISCUSSION TITLE
Zoom to group

### DISCUSSION THREAD

**Discussion by gi921120** [opengisch/QField#2739] [Q&A]
Hi there, 

Is there a way to zoom to the group in the app?

Thank you 

Giovanny

**Comment by VxTedxV** [comment]
You can zoom to a layer by long pressing on its name and choose "zoom to layer" but I think you can't zoom to a group of layer (yet ?).

**Comment by suricactus** [comment]
@gi921120 check https://github.com/opengisch/QField/pull/2898 :)
